packs/nitropack.js

```diff
@@ -0,0 +1,41 @@
+  /** Additional description of a Lighthouse audit for a third-party framework called `NitroPack`. This is displayed after a user expands the section to see more. No character length limits. `Image Optimization` is the name of a feature and becomes link text to additional documentation.*/
+  'uses-optimized-images': 'Automatically compress, optimize, and convert your images into WebP by enabling the [`Image Optimization`](https://support.nitropack.io/hc/en-us/articles/14177271695121-How-to-serve-images-in-next-gen-formats-using-NitroPack) setting.',
+  /** Additional description of a Lighthouse audit for a third-party framework called `NitroPack`. This is displayed after a user expands the section to see more. No character length limits. `Gzip compression` is the name of a feature and becomes link text to additional documentation.*/
+  'uses-text-compression': 'Use [`Gzip compression`](https://support.nitropack.io/hc/en-us/articles/13229297479313-Enabling-GZIP-compression) in NitroPack to reduce the size of the files that are sent to the browser. This includes text files such as HTML, CSS, and JavaScript. The smaller file size allows for quicker transmission of data from the server to the client, resulting in faster page load times.',
+  /** Additional description of a Lighthouse audit for a third-party framework called `NitroPack`. This is displayed after a user expands the section to see more. No character length limits. `Adaptive Image Sizing` is the name of a feature and becomes link text to additional documentation.*/
+  'uses-responsive-images': 'Enable [`Adaptive Image Sizing`](https://support.nitropack.io/hc/en-us/articles/10123833029905-How-to-Enable-Adaptive-Image-Sizing-For-Your-Site) to preemptively optimize your images and make them match the dimensions of the containers they’re displayed in across all devices. This will also help reduce the amount of unnecessary bytes downloaded.',
+  /** Additional description of a Lighthouse audit for a third-party framework called `NitroPack`. This is displayed after a user expands the section to see more. No character length limits. `Instant Load` is the name of a feature and becomes link text to additional documentation.*/
+  'server-response-time': 'Improve server response time and optimize perceived performance by activating [`Instant Load`](https://support.nitropack.io/hc/en-us/articles/16547340617361). This powerful feature prefetches your website\'s frequently accessed pages, ensuring an instant loading experience for your valued users.',
+  /** Additional description of a Lighthouse audit for a third-party framework called `NitroPack`. This is displayed after a user expands the section to see more. No character length limits. `HTML Lazy Load` is the name of a feature.*/
+  'dom-size': 'Contact your account manager to enable `HTML Lazy Load`. Configuring it will prioritize and optimize your page rendering performance.',
```

Comment:
Is there a documentation link for this? I would like to know how this affects DOM size.

Reply:
@ivailohristov are there no docs for this features? If not can you add some more details on what this feature does to improve the DOM size.

Reply:
Hi @adamraine, we didn't actually have docs on this. But now we do :smiley: 
Feel free to check it out and let me know if you have any questions